### ISSUE TITLE
beg for version 2.3.3 release

### ISSUE THREAD

**Issue by Kewings** [PyFPDF/fpdf2#133]
hi team, 

would you please help to release version 2.3.3 with img in td fixed?

thanks.

**Comment by Lucas-C** [comment]
I plan to perform a new release once the 2 currently opened PRs are merged.

Checking with the co-maintainer of this repo : @alexp1917 do you agree with that plan ? 🙂

**Comment by alexp1917** [comment]
Sounds good to me, I've been buried with stuff lately, but looks like
you've got a handle on this

**Comment by alexp1917** [comment]
@Lucas-C let me know if i should go ahead and do this release. im getting caught up on all the issues. looks like you have been busy here, but i dont see what to wait on, or if anything else is in progress - please let me know. otherwise i will get to this tonight hopefully.

**Comment by Lucas-C** [comment]
I'm waiting on a review from you on https://github.com/PyFPDF/fpdf2/pull/130 but I do not want to hurry you at all.

This PR introduces new methods so I thought it'd be nice to have your feedback on it.

Tell me once you approve to merge it 😉

**Comment by Lucas-C** [comment]
New release published: https://github.com/PyFPDF/fpdf2/releases/tag/2.3.3
